lib/src/tabs-nav/Tab.tsx

```diff
@@ -103,16 +117,18 @@ const Tab = styled.a<TabStyleProps>`
   ${(props) =>
     !props.disabled &&
     `:hover {
-    background: #0000000D;
+    background: ${props.theme.unselectedBackgroundColor};
   }
```

Comment:
Change this token to `hoverBackgroundColor`.

Reply:
done